tests/unit/test_file.py

```diff
@@ -437,29 +437,35 @@ def test_multi_publications(self):
 
 class SubjectTest(TestCase):
     def setUp(self):
-        self.subject = Subject(age='P90D',
-                               description='An unfortunate rat',
-                               genotype='WT',
-                               sex='M',
-                               species='Rattus norvegicus',
-                               subject_id='RAT123',
-                               weight='2 kg',
-                               date_of_birth=datetime(2017, 5, 1, 12, tzinfo=tzlocal()),
-                               strain='my_strain')
+        self.subject = Subject(
+            age='P90D',
```

Comment:
Do we need a backward compatibility test for the case when age reference is missing?

https://github.com/NeurodataWithoutBorders/pynwb/tree/dev/tests/back_compat

Reply:
Yes, we should add a test just in case.